analyzer/src/main/resources/META-INF/services/org.ossreviewtoolkit.analyzer.PackageManagerFactory

```diff
@@ -1,9 +1,10 @@
-org.ossreviewtoolkit.analyzer.managers.DotNet$Factory
+org.ossreviewtoolkit.analyzer.managers.Carthage$Factory
+org.ossreviewtoolkit.analyzer.managers.CocoaPods$Factory
+org.ossreviewtoolkit.analyzer.managers.Conan$Factory
```

Comment:
These must not be added back here.

Reply:
@sschuberth The latest push addresses this mixup.